src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp

```diff
@@ -63,14 +63,15 @@ inline arm_compute::DataType precisionToAclDataType(InferenceEngine::Precision p
 /**
 * @brief Return ComputeLibrary DataLayout that corresponds to MemoryDecs layout
 * @param desc MemoryDecs from which layout is retrieved
+* @param treatAs4D the flag that treats MemoryDecs as 4D shape
 * @return ComputeLibrary DataLayout or UNKNOWN if MemoryDecs layout is not mapped to DataLayout
 */
-inline arm_compute::DataLayout getAclDataLayoutByMemoryDesc(MemoryDescCPtr desc) {
+inline arm_compute::DataLayout getAclDataLayoutByMemoryDesc(MemoryDescCPtr desc, bool treatAs4D = false) {
```

Comment:
Actually this change looks like workaround.
Looking on ACL my impression is that they are using NCHW/NHWC layouts to describe the tensor even if it has lower rank than 4. 
So my proposal is to always return NCHW/NHWC for ranks 1-4. Other logic should remains untouched.

Reply:
Fixed